todos/src/components/FilterButtons.js

```diff
@@ -1,13 +1,55 @@
-import React from 'react'
+import React from "react";
+import { setFilter } from "actions/index";
+import { useDispatch, useSelector } from "react-redux";
+import { ALL, INCOMPLETE, COMPLETE } from "constants";
+
+const FilterButtons = () => {
+  const dispatch = useDispatch();
+  const filter = useSelector((state) => state.filterReducer.filter);
+  const listLength = useSelector((state) => state.todoReducers.list).length;
+
+  const handleAllTodos = () => {
+    dispatch(setFilter(ALL));
+  };
+  const handleIncompleteTodos = () => {
+    dispatch(setFilter(INCOMPLETE));
+  };
+  const handleCompleteTodos = () => {
+    dispatch(setFilter(COMPLETE));
+  };
 
-function FilterButtons() {
   return (
-    <div className="btn__progress_button">
-      <button>All</button>
-      <button>Incomplete</button>
-      <button>Complete</button>
+    <div
+      className={`btn__progress_button ${
+        listLength === 0 ? "btn__progress_button--disable" : null
```

Comment:
please use classnames npm package here

Reply:
@ray-wtag Bhai, resolved [here](https://github.com/fza-wtag/todo-app-react/pull/9#discussion_r1111192324) in another PR.